src/Argu/Utils.fs

```diff
@@ -131,19 +145,25 @@ type IDictionary<'K,'V> with
         else None
 
 
+#if !CORE_CLR
 let currentProgramName = lazy(System.Diagnostics.Process.GetCurrentProcess().MainModule.ModuleName)
+#else
+// error FS0039: The value, constructor, namespace or type 'Process' is not defined
+let currentProgramName = lazy("wtf")
```

Comment:
I can look again, but I couldn't make if work at the time. But you are right it shouldn't stay this way :)